### DISCUSSION TITLE
OpenROAD GUI Leaf instance browser

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenROAD#4677] [General]
With latest OpenROAD GUI, Leaf Instance > Combinational not showing any instance list.
![Missing_leaf_instance_list](https://github.com/The-OpenROAD-Project/OpenROAD/assets/85787913/825e771d-7eef-42c7-9779-89bb23263e48)

But previous OpenROAD had this feature to trace all leaf instance list.
![with_instance_list](https://github.com/The-OpenROAD-Project/OpenROAD/assets/85787913/4253f57c-7360-4978-babf-f773ecc310b7)

Is that feature moved under some other options or removed it?

@maliberty @vvbandeira

**Comment by maliberty** [comment]
The listing of instances was removed as for large designs it becomes far too slow and isn't that helpful.  See https://github.com/The-OpenROAD-Project/OpenROAD/issues/4629  What is your use case?

**Reply by vijayank88** [reply]
Yes, I agree for complex design.
But for beginners it is good to keep cell list in Hier. browser itself.
Is that possible to add any switch to build by user for enable or disable this feature?
By default keep it disable, on need basis we can enable during the build.

**Reply by maliberty** [reply]
You can control visibility by type through the display controls.  It would only be useful if you want to see the cells associated with a particular module.  That doesn't seem like something a beginner needs.  Do you think differently?
